pgjdbc/src/main/java/io/materialize/jdbc/TypeInfoCache.java

```diff
@@ -103,18 +103,28 @@
   static {
     typeAliases = new HashMap<String, String>();
     typeAliases.put("smallint", "int2");
+
+    typeAliases.put("i32", "int4");
     typeAliases.put("integer", "int4");
     typeAliases.put("int", "int4");
-    typeAliases.put("bigint", "int8");
-    typeAliases.put("float", "float8");
-    typeAliases.put("boolean", "bool");
-    typeAliases.put("decimal", "numeric");
 
-    typeAliases.put("i32", "int4");
     typeAliases.put("i64", "int8");
+    typeAliases.put("bigint", "int8");
+
     typeAliases.put("f32", "float4");
+    typeAliases.put("real", "float4");
+
     typeAliases.put("f64", "float8");
+    typeAliases.put("double precision", "float8");
+    typeAliases.put("double", "float8");
+    typeAliases.put("float", "float8");
+
+    typeAliases.put("text", "varchar");
     typeAliases.put("string", "varchar");
+
+    typeAliases.put("boolean", "bool");
+    typeAliases.put("decimal", "numeric");
+    typeAliases.put("json", "jsonb");
```

Comment:
these (json/jsonb) aren't technically identical but they should behave similarly enough for metabase purposes so it's probably fine?

Reply:
yeah--it's definitely sort of weird! but we do technically list it as an alias [here](https://materialize.com/docs/sql/types/#built-in-types).